hack/build-antrea-windows-all.sh

```diff
@@ -75,15 +89,30 @@ WIN_BUILD_TAG=$(echo $GO_VERSION $CNI_BINARIES_VERSION $NANOSERVER_VERSION| md5s
 
 echo "WIN_BUILD_TAG=$WIN_BUILD_TAG"
 
+docker pull mcr.microsoft.com/windows/servercore:$NANOSERVER_VERSION
```

Comment:
Why shall we pull these basic Windows images every time? Will they be cached on the image building host?

Reply:
This script can be used in github action job, which creates a new vm for building each time.